### DISCUSSION TITLE
请问DatetimePicker 组件重构成了PickerGroup + TimePicker、DatePicker之后，这种怎么实现？

### DISCUSSION THREAD

**Discussion by Homebrew1** [youzan/vant#13051] [Q&A]
### 这种怎么实现？

![image](https://github.com/user-attachments/assets/5a8a3aa6-0102-4fe3-bc05-fa8882b9655e)
PickerGroup 是多个tab栏，但是我要DatePicker在前，TimePicker在后，并且DatePicker格式化成 月日加星期。
而不是tab栏

### 你期望的 API 是什么样子的？

<van-picker-group
        title="预约日期"
        :tabs="['选择日期', '选择时间']"
        @confirm="state.showAppointment = !state.showAppointment"
        @cancel="state.showAppointment = !state.showAppointment"
      >
        <van-date-picker
          v-model="state.appointDate"
          :min-date="minDate"
          :max-date="maxDate"
        />
        <van-time-picker :filter="filter" v-model="state.appointTime" />
      </van-picker-group>

**Comment by inottn** [comment]
这种场景只能用 Picker 组件封装一下了
